app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsController.scala

```diff
@@ -526,6 +552,61 @@ class AcquisitionDetailsController @Inject() (
     }
   }
 
+  def shouldUseRebase(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withFillingOutReturnAndAcquisitionDetailsAnswers(request) {
+      case (_, _, draftReturn, answers) =>
+        withAssetTypeAndResidentialStatus(draftReturn, answers) {
+          case (assetType, wasAUkResident) =>
```

Comment:
if we can determined that they're a non uk resident can we not kick them out here to check your answers? We know they shouldn't be seeing this page